code/modules/requisitions/_supply_export.dm

```diff
@@ -1,22 +1,7 @@
 /atom/movable/proc/supply_export()
-	return 0
-
-
-/obj/structure/closet/crate/supply_export()
-	. = 3
-	SSpoints.supply_points += .
-
-
-/mob/living/carbon/xenomorph/supply_export()
-	switch(tier)
-		if(XENO_TIER_ZERO)
-			. = 1
-		if(XENO_TIER_ONE)
-			. = 5
-		if(XENO_TIER_TWO)
-			. = 10
-		if(XENO_TIER_THREE)
-			. = 30
-		if(XENO_TIER_FOUR)
-			. = 50
+	. = 0
+	for(var/datum/supply_export in GLOB.exports_types)
+		var/datum/supply_export/E = supply_export
+		if(type == E.export_obj)
+			. = E.cost

```

Comment:
This was intentionally changed so we could balance stuff on Crash separately, as xeno bodies are much cheaper to come back. Do please revert this change. Let's start small and if xenos are way too cheap we can always increase their value.

Reply:
Putting it back as it was before